pkg/auth/auth.go

```diff
@@ -96,6 +96,7 @@ type Action struct {
 // Checker for the exported funcs
 type Checker interface {
 	Validate() error
```

Comment:
is Validate being used anymore now?

Reply:
No, it's no longer used. I'll remove that too.